src/Framework/PaymentGateways/EventHandlers/DonationAbandoned.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Give\Framework\PaymentGateways\EventHandlers;
+
+use Exception;
+use Give\Donations\ValueObjects\DonationStatus;
+use Give\Framework\PaymentGateways\Actions\UpdateDonationStatus;
+
+/**
+ * @unreleased
+ */
+class DonationAbandoned
+{
+    /**
+     * @unreleased
+     * @throws Exception
+     */
+    public function __invoke(string $gatewayTransactionId, string $message = '', $skipRecurringDonations = false)
```

Comment:
I just noticed these event handlers only receive a `$gatewayTransactionId` rather than a Donation ID.  I think i lost sight of the context here and could use your help explaining:
1. What are we considering to be an `Event`?
2. Are EventHandlers intended to be used directly by action scheduler / webhook?

Reply:
@glaubersilva thanks for the clarification, i'm tracking now 😄 

Yes lets go with your suggestion and move these into a new `Webhooks` folder to give these EventHandlers more context - i'm sure there will be more webhook related functionality we can add to that folder in the future like validation helpers and url generators.

Reply:
Resolved in a5c3f58f4c06130591d0e3717b4e9b2c3b8fdde7